inscriptions/ISic001152.xml

```diff
@@ -190,43 +195,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140637">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    
-                    <expan>
-                        <abbr>Θε</abbr>
-                        <ex>οῖς</ex>
-                    </expan>
-                    ∙
-                    <expan>
-                        <abbr>Κα</abbr>
-                        <ex>ταχθονίοις</ex>
-                    </expan>
-                    .<lb n="2"/>Πονπωνία<lb n="3"/>Μάξιμα<lb n="4"/>
-                    <expan>
-                        <abbr>ἔζ</abbr>
-                        <ex>ησεν</ex>
-                    </expan>
-                    ∙
-                    <expan>
-                        <abbr>ἔτ</abbr>
-                        <ex>η</ex>
-                    </expan>
-                    ∙ μʹ.<lb n="5"/>Κ<supplied reason="omitted">λ</supplied>(αύδιος) Μ<supplied reason="omitted">
-                    έ</supplied>λισσος<lb n="6"/>τῇ ἰδίᾳ<lb n="7"/>συ
-                    <supplied reason="lost">μ</supplied>
-                    <unclear>β</unclear>ίῳ.
-                    <lb n="8"/>
+                    <lb n="1"/><expan><abbr>Θ<choice><corr>ε</corr><sic>σ</sic></choice></abbr><ex>οῖς</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Κα</abbr><ex>ταχθονίοις</ex></expan>.
+                    <lb n="2"/><name>Πονπωνία</name>
+                    <lb n="3"/><name>Μάξιμα</name>
+                    <lb n="4"/><expan><abbr>ἔζ</abbr><ex>ησεν</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>ἔτ</abbr><ex>η</ex></expan> <g ref="#interpunct">·</g> <num value="40">μ</num>.
+                    <lb n="5"/><name><expan><abbr>Κ<choice><corr>λ</corr><sic>α</sic></choice></abbr><ex>αύδιος</ex></expan></name> <name>Μ<choice><corr>έ</corr><sic>ο</sic></choice>λισσος</name>
+                    <lb n="6"/>τῇ ἰδίᾳ
+                    <lb n="7"/><orig>συ</orig><supplied reason="lost">μ</supplied><unclear>β</unclear>ίῳ.
                 </ab>
             </div>
-            <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+            <div type="apparatus" resp="#MH">
+                <p>L. 7: Gualt. reads ΣΥ...ΡΙΩ</p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#MH">
+                <p>To the gods of the Underworld. Pomponia Maxima lived 40 years. Klaudios Melissos (made this) for his wife.</p>
```

Comment:
I'd go for something like 'life partner', rather than wife, to capture the distinction of 'sunbios'